apps/order/schema.py

```diff
@@ -96,6 +98,79 @@ class Meta:
         filterset_class = OrderFilterSet
 
 
+class OrderTypeForStatistic(DjangoObjectType):
+    total_quantity = graphene.Int()
+
+    class Meta:
+        model = Order
+        fields = ('id', 'order_code', 'total_price')
+
+    def resolve_total_quantity(root, info, **kwargs):
+        return root.book_order.aggregate(Sum('quantity'))['quantity__sum']
+
+
+stat_to = datetime.date.today()
+stat_from = stat_from = stat_to - datetime.timedelta(30)
```

Comment:
Why is this outside? This will be set on application initialization and wouldn't change until the application is killed.

Reply:
@thenav56  I added these to reuse, but moved inside now